liquid-compiler/src/grammar.pest

```diff
@@ -0,0 +1,56 @@
+WHITESPACE = _{" "}
+LINE_OR_SPACE = _{ NEWLINE | WHITESPACE }
+LiquidFile = ${ SOI ~ Element* ~ EOI }
+
+// Element-level parsing
+Element = _{ Expression | Tag | Raw } 
+
+TagStart = _{ (LINE_OR_SPACE* ~ "{%-") | "{%" }
+TagEnd =   _{ ("-%}" ~ LINE_OR_SPACE*) | "%}" }
+ExpressionStart = _{ (LINE_OR_SPACE* ~ "{{-") | "{{" }
+ExpressionEnd =   _{ ("-}}" ~ LINE_OR_SPACE*) | "}}" }
```

Comment:
I didn't notice in the Ruby liquid, does it support `{{-` and friends for stripping whitespace?

Reply:
Apparently it is
https://shopify.github.io/liquid/basics/whitespace/

Reply:
I'm not sure about Ruby liquid, but it does indeed happen in the old parser of liquid-rust and also in [liquidjs](https://www.npmjs.com/package/liquidjs). I think there is no need to remove that, as it may be helpful.

Reply:
Oh it seems you already found that out. I didn't refresh the page, so I didn't see the new messages.